ntp-ntpv5.xml

```diff
@@ -1407,12 +1409,63 @@ Tx  | 0  |    | t3'|      | 0  |    | t3 |      | 0  |    |t11'|
     </section>
 
     <section anchor="Security" title="Security Considerations">
-      <t>The corrections provided by network devices in the Correction
+      <t>The security considerations of time protocols in general are 
+	      discussed in <xref target="RFC7384"/>. A successful attack on the 
+	      time distribution can result in one of three outcomes: Denial of 
+	      Service (DoS), reduced accuracy, or obtaining incorrect time.
+	      NTP can be compromised through various methods, such as altering 
+	      or delaying NTP messages during transit, injecting malicious NTP 
+	      messages or replaying valid ones, or impersonating an NTP server.</t>
+		
+      <t>The MAC extension field can be used to provide integrity protection, 
+	      thus mitigating in-transit NTP message modification and malicious 
+	      packet injection.</t>
+		
+      <t>Using NTS with NTPv5 provides enhanced security properties, including 
+	      server identity verification, replay protection and secure key 
```

Comment:
Replay protection is already provided by the NTP client-server mode, it's not a new feature of NTS. Maybe "improved replay protection" would work better?

Reply:
Yes, sounds good.